### ISSUE TITLE
"phpstormFormatter.styleGuidePath" is not working

### ISSUE THREAD

**Issue by Kaimura** [animus-coop/vscode-phpstorm-formatter#8]
While the extension at least detects my 2018 phpstorm verion it somehow cannot detect the XML file which is giving the rules for formatting like so:
`-<codeStyleSettings language="PHP">

<option name="BLANK_LINES_AFTER_PACKAGE" value="1"/>

<option name="ALIGN_MULTILINE_PARAMETERS" value="false"/>

<option name="CALL_PARAMETERS_WRAP" value="1"/>
...
...`
I placed the code style xml file in the same folder where my phpstorm exe lies (Program Files\\JetBrains\\PhpStorm 2018.1.6\\bin) So my path for the xml file is Program Files\\JetBrains\\PhpStorm 2018.1.6\\bin\\CodeStyle.xml .. I know that this xml file works on phpstorm itself!

But everytime I use "Fomat Document with..." and choose phpstorm fomatter it immediately delivers a "Command 'Format Document With...' resulted in an error." and "Failed to format document" message... I also tried leaving out .xml and also just specifying the folder but nothing works.. (phpstorm is closed as well btw)

edit: okay apparently xml code is being ignored.. does not matter for this question, anyway

**Comment by soymanuvalle** [comment]
Thanks for reporting, I'll do some testing regarding that and will get back to you.

**Comment by soymanuvalle** [comment]
Hello @Kaimura. I've tested the extension on Windows and I've found out the path wasn't being parsed correctly for windows. I've fixed this on the latest update.

I've tested the style guide with the following configuration and worked without problems:
![image](https://user-images.githubusercontent.com/9582420/79136698-d07aff00-7d87-11ea-8b3b-1d37f2fb5d6a.png)


Also, on my side PHPStorm crashed several times so I had to increase the memory heap limit of the VM: https://www.jetbrains.com/help/idea/increasing-memory-heap.html (vscode-phpstorm-formatter uses the native PHPStorm format script on the background)
